### PR TITLE
V1.4.3

### PR DESCRIPTION
**Pull request by 01000011-00110111**

### CODE REVIEW

File: setup.py
Comment (01000011-00110111) [inline setup.py:1]: please add this file back before we continue

**Review by 01000011-00110111 — COMMENTED** [review]
Fix these and ill add the message feature soon if not then in 1.4.4

File: main.py [hunk 1]
```diff
@@ -796,6 +802,9 @@ def handle_heartbeat(status):
 
         if userid in user_connections:
             update_userlist(socketid, {"status": status}, userid)
+            room = Chat.create_or_get_chat(roomid)
+            if socketid not in room.sids:
+                room.sids.append(socketid)
```
Comment (01000011-00110111) [inline main.py:807]: before line 807 add 
```py
socketids[userid] = (socketid)
```

**Review by CastyiGlitchxz — APPROVED** [review]
Everything is fine.

**Pull request merged**